app/assets/javascripts/discourse/app/widgets/topic-admin-menu.js

```diff
@@ -76,33 +76,44 @@ createWidget("topic-admin-menu-button", {
 
   showAdminMenu(e) {
     this.state.expanded = true;
-    let $button;
+    let button;
 
     if (e === undefined) {
-      $button = $(".keyboard-target-admin-menu");
+      button = document.getElementsByClassName(".keyboard-target-admin-menu");
```

Comment:
This returns `NodeList` (an empty one, because of the dot 😛) while you seem to require a single Element:

```suggestion
      button = document.querySelector(".keyboard-target-admin-menu");
```